github_poster/loader/jike_loader.py

```diff
@@ -48,13 +48,14 @@ def add_loader_arguments(cls, parser, optional):
             "--jike_user_id",
             dest="user_id",
             type=str,
+            required=True,
             help="The user id of jike",
         )
         parser.add_argument(
             "--jike_cookie",
             dest="jike_cookie",
             type=str,
-            required=optional,
+            required=True,
```

Comment:
Change here maybe we can not support Multiple or summary.

Reply:
ok, I will change back.